rmw_zenoh_cpp/src/detail/rmw_node_data.cpp

```diff
@@ -210,7 +207,7 @@ bool NodeData::create_sub_data(
   }
 
   auto sub_data = SubscriptionData::make(
-    std::move(session),
+    session,
```

Comment:
Why do we move the `std::shared_ptr<GraphCache>` but not the `std::shared_ptr<zenoh::Session> session`?